ClientApp/src/Pages/GettingStarted/GettingStarted.tsx

```diff
@@ -0,0 +1,104 @@
+
+import React, { useState } from 'react';
+import LocationData from './LocationData.json';
+import './GettingStarted.scss';
+
+import { LocationSummary } from './LocationSummary';
+import ReactWeather, { useOpenWeather } from 'react-open-weather';//ignore red-line, it still works
+
+enum Locations {
+    Scotland,
+    Canada,
+    Australia
+}
+
+function GettingStarted() {
+    const [selectedLocation, setSelectedLocation] = useState(Locations.Scotland);
+
+    let x;
+    let y;
+    let loc;
+
+    const latScotland = '57.7207794189';
+    const LonScotland = '-5.6862931252';
+    const LatCanada = '53.135509';
+    const LonCanada = '-57.660435';
+    const LatAustralia = '-20.917574';
+    const LonAustralia = '142.702789';
+
+    if (selectedLocation === Locations.Canada) {
+        x = LatCanada;
+        y = LonCanada;
+        loc = " NewFoundland, Canada";
+    }
+    else if (selectedLocation === Locations.Scotland) {
+        x = latScotland;
+        y = LonScotland;
+        loc = "Gairloch, Scotland"
+    }
+    else if (selectedLocation === Locations.Australia) {
+        x = LatAustralia;
+        y = LonAustralia;
+        loc = "Queensland, Australia"
+    }
+
+    const { data, isLoading, errorMessage } = useOpenWeather({
+        key: '43887186b07bb5deaad592918ae44878',
+        lat: x,
+        lon: y,
+        lang: 'en',
+        unit: 'metric', // values are (metric, standard, imperial)
+    });
+
+    return (
+        <div>
+            <div className="getting_started">
+
+                <div className='top-page-break'></div>
+
+                <h2>Getting Started: Find the Perfect Whalespotting Location</h2>
+                <div className="places">
+                    <h3 /*data-testid={"ScotlandButton"}*/ className={selectedLocation === Locations.Scotland ? 'selected-icon' : 'unselected-icon'} onClick={() => setSelectedLocation(Locations.Scotland)}>Scotland</h3>
+                    <h3 /*data-testid={"CanadaButton"}*/ className={selectedLocation === Locations.Canada ? 'selected-icon' : 'unselected-icon'} onClick={() => setSelectedLocation(Locations.Canada)}>Canada</h3>
+                    <h3 /*data-testid={"AustraliaButton"}*/ className={selectedLocation === Locations.Australia ? 'selected-icon' : 'unselected-icon'} onClick={() => setSelectedLocation(Locations.Australia)}>Australia</h3>
```

Comment:
all these `h3`s look like they are being used as buttons... (or perhaps some kind of form input?)
If so... why not just make them buttons?

That way:
- styling for if they are pressed is easier
- its better for accessibility
- the browser understands what's happening, so will automatically give it sensible hover states, keyboard interaction etc etc